cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts

```diff
@@ -12,6 +12,22 @@ describe('Pods', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, ()
     cy.login();
   });
 
+  describe('List small', { tags: ['@vai', '@adminUser'] }, () => {
+    it('pagination is hidden', () => {
```

Comment:
is there a reason this test is in its own describe block?

Reply:
Its behavior is really brittle when running it as part of the other block, I didn't find a good reason for that after investigating, it just fails a good percentage of runs as if the intercept is not caught properly.